### PR TITLE
Tasks/#58 release v0.14.0

### LINKED ISSUE #58: should release v0.14.0
v0.14.0 is ready to be released

### PR DESCRIPTION
**Pull request by hoatle**
connect #58 

This is the final changes for v0.14.0.

We followed the release steps here https://github.com/teracyhq/flask-classful/blob/develop/docs/release.rst

@hjc please take a look to see if everything is ok before I'm tagging and publishing to the PyPi.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+
+    - Bugfixes:
+        * The behavior of the trailing slash should be more intuitive and consistent #47
+        * `base_args` should not be overriden when `route_base` is set #50
+
+    - Tasks:
+        * Keep the original license #51
+        * Fix docs publishing #61
+        * Breaking change re: FlaskView.decorators missing in CHANGELOG (for v0.11.0) #49
+        * Should update docs for v0.14.0 #60 #73
```
Comment (hjc) [inline CHANGELOG.md]: I don't think this line needs to be included, as it should be obvious.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+        * Docker workflow along with teracy-dev for better dev setup #32 #63 #66 #67
+
+    - Bugfixes:
+        * The behavior of the trailing slash should be more intuitive and consistent #47
+        * `base_args` should not be overriden when `route_base` is set #50
+
+    - Tasks:
+        * Keep the original license #51
+        * Fix docs publishing #61
+        * Breaking change re: FlaskView.decorators missing in CHANGELOG (for v0.11.0) #49
```
Comment (hjc) [inline CHANGELOG.md]: Can we make this a larger section? Like a `Breaking Changes` section all on it's own so it's obvious to everyone?

File: CHANGELOG.md (hunk 1 above)
Comment (hoatle) [reply CHANGELOG.md]: removed

File: CHANGELOG.md (hunk 2 above)
Comment (hoatle) [reply CHANGELOG.md]: It's expected to have some breaking changes before v1.0.0 for faster innovation (following semantic versioning).
I agree, it's great to have a `Breaking Changes` section, maybe you could help with that section content?

**Comment by hoatle** [comment]
This is good enough to me, merged then.

**Comment by hoatle** [comment]
after this is merged, I'm going to release v0.14.0.

File: CHANGELOG.md (hunk 2 above)
Comment (hjc) [reply CHANGELOG.md]: Totally agreed that we gotta break some stuff to innovate faster.

I'll whip up something really quick here and open a PR.

Apologies for the silence, got sick over the weekend.

**Comment by hjc** [comment]
👍

**Pull request merged**